### DISCUSSION TITLE
MD Files do not open

### DISCUSSION THREAD

**Discussion by banda64** [microsoft/azuredatastudio#18144] [Q&A]
Hello All,
I am new to VS Code.
I have been facing an issue, but could not find solution on net.
So thought someone in the community can let me know if its my setup issue or my corporate network issue.

In extension information page there are 4 tabs.
1. Details (README.md)
2. Feature Contributions
3. Changelog (CHANGELOG.md)
4. Runtime Status

But when I click the details tab it does not display anything.
But when I clicked Feature contributions it displays fine.
When I click Changelog couple of loading dots appear and does not shown anything
Runtime Status works fine (I guess), its empty with some border.

I could understand that my VCSCode is unable to display *.md files.
But  could not understand what I missed in setup that is causing this issue.
Tried on couple of different PC's and I could see same result.
I have checked the Readme.md file of extension and its not empty it has information.

Note: I had to use proxy to download and install extension in my corp environment.
So Proxy is required to reach market place.

Below is my VS Code info
Version: 1.63.2 (user setup)
Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3
Date: 2021-12-15T09:40:02.816Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19044

Any suggestions would be very much helpful.

Thanks
Naresh Kumar
![VSCode_Error](https://user-images.githubusercontent.com/63230785/150824412-344a6394-b554-4bc2-af0f-1708aef976ba.JPG)

**Comment by Charles-Gagnon** [comment]
Sorry to hear you're having problems @banda64! This is the repo for Azure Data Studio (a tool built on VS Code but it's a different product). 

I would suggest opening an issue on their repo and hopefully someone there can help you out! https://github.com/Microsoft/vscode
